lib/ae_page_objects/version.rb

```diff
@@ -1,3 +1,3 @@
 module AePageObjects
-  VERSION = '2.1.0'.freeze
+  VERSION = '2.0.1'.freeze
```

Comment:
Why was the version number decreased?  Given the API changes we'll need a major version number, making this `3.0.0`